arelle/plugin/validate/NL/rules/br_kvk.py

```diff
@@ -282,3 +283,41 @@ def rule_br_kvk_4_16(
                 resubmissionConceptQname=resubmissionConceptQname,
                 conceptQname=conceptQname,
             )
+
```

Comment:
```suggestion


```
Nit: my IDE suggests two blank lines in this context. [Underlying flake8 rule](https://www.flake8rules.com/rules/E302.html)

Reply:
I hate flake8 more than anything in the world, well maybe not as much as the Griz, but thats neither here or there. I do however, agree with Aaron's sentiment on this. 2 lines please.